src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -1466,6 +1467,82 @@ namespace OpenLoco::Ui::WindowManager
         invalidateWidget(window->type, window->number, widgetIndex);
     }
 
+    static bool isStepperGroup(Window& w, WidgetIndex_t index, WidgetType buttonType)
+    {
+        const auto& widgets = w.widgets;
+
+        if (widgets[index].type != WidgetType::textbox && widgets[index].type != WidgetType::wt_3)
```

Comment:
what's `wt_3`, do we know enough to name it (in another pr of course)

Reply:
It's used in various ways -- see below. This is making it harder to name. Let's move the discussion to #708.
```
$ grep -R 'wt_3' src/
src//OpenLoco/src/Widget.cpp:            case WidgetType::wt_3:
src//OpenLoco/src/Ui/Dropdown.cpp:            makeWidget({ 0, 0 }, { 1, 1 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Window.h:        wt_3,
src//OpenLoco/src/Windows/TitleLogo.cpp:        makeWidget({ 0, 0 }, kWindowSize, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/PlayerInfoPanel.cpp:        makeWidget({ 0, 0 }, { 140, 29 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/PlayerInfoPanel.cpp:        makeWidget({ 2, 2 }, { 136, 25 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/TerraForm.cpp:            makeWidget({ 33 + 16, 45 }, { 64, 44 }, WidgetType::wt_3, WindowColour::secondary, ImageIds::tool_area, StringIds::tooltip_clear_area),
src//OpenLoco/src/Windows/TerraForm.cpp:            makeWidget({ 33 + 16, 45 }, { 64, 44 }, WidgetType::wt_3, WindowColour::secondary, ImageIds::tool_area, StringIds::tooltip_adjust_land_tool),
src//OpenLoco/src/Windows/TerraForm.cpp:            makeWidget({ 33 + 16, 45 }, { 64, 44 }, WidgetType::wt_3, WindowColour::secondary, ImageIds::tool_area, StringIds::tooltip_adjust_water_tool),
src//OpenLoco/src/Windows/Error.cpp:            makeWidget({ 0, 0 }, { 200, 42 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/Error.cpp:            makeWidget({ 0, 0 }, { 250, 70 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/Error.cpp:            makeWidget({ 3, 3 }, { 64, 64 }, WidgetType::wt_3, WindowColour::secondary),
src//OpenLoco/src/Windows/MapToolTip.cpp:        makeWidget({ 0, 0 }, { 200, 30 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/ToolbarBottomEditor.cpp:        makeWidget({ 0, 0 }, { 200, 34 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/ToolbarBottomEditor.cpp:        makeWidget({ 440, 0 }, { 200, 34 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/ToolbarBottomEditor.cpp:        self.widgets[widx::next_frame].type = WidgetType::wt_3;
src//OpenLoco/src/Windows/ToolbarBottomEditor.cpp:            self.widgets[widx::previous_frame].type = WidgetType::wt_3;
src//OpenLoco/src/Windows/ToolTip.cpp:        makeWidget({ 0, 0 }, { 200, 32 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/News/Ticker.cpp:        makeWidget({ 0, 0 }, { 111, 26 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/News/News.cpp:            commonWidgets(360, 117, WidgetType::wt_3),
src//OpenLoco/src/Windows/Tutorial.cpp:        makeWidget({ 0, 0 }, kWindowSize, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/Construction/Construction.h:        makeWidget({ 0, 41 }, { frameWidth, frameHeight - 41 }, WidgetType::wt_3, WindowColour::secondary),                                                          \
src//OpenLoco/src/Windows/Construction/OverheadTab.cpp:        makeWidget({ 35, 110 }, { 66, 66 }, WidgetType::wt_3, WindowColour::secondary),
src//OpenLoco/src/Windows/Construction/OverheadTab.cpp:            self.widgets[widx::image].type = WidgetType::wt_3;
src//OpenLoco/src/Windows/Construction/StationTab.cpp:        makeWidget({ 35, 60 }, { 68, 68 }, WidgetType::wt_3, WindowColour::secondary),
src//OpenLoco/src/Windows/Construction/ConstructionTab.cpp:            window->widgets[widx::construct].type = WidgetType::wt_3;
src//OpenLoco/src/Windows/Construction/ConstructionTab.cpp:            window->widgets[widx::construct].type = WidgetType::wt_3;
src//OpenLoco/src/Windows/TimePanel.cpp:        makeWidget({ 0, 0 }, { 140, 29 }, WidgetType::wt_3, WindowColour::primary),                                                                                                   // 0,
src//OpenLoco/src/Windows/TimePanel.cpp:        makeWidget({ 2, 2 }, { 136, 25 }, WidgetType::wt_3, WindowColour::primary),                                                                                                   // 1,
src//OpenLoco/src/Windows/DragVehiclePart.cpp:        makeWidget({ 0, 0 }, { 150, 60 }, WidgetType::wt_3, WindowColour::primary),
src//OpenLoco/src/Windows/ScenarioSelect.cpp:        makeWidget({ 0, 48 }, { 610, 364 }, WidgetType::wt_3, WindowColour::secondary),
```